tests/workspace/test_cli.py

```diff
@@ -47,7 +46,7 @@ def test_export_workspace_cli(workspace_api_mock, tmpdir):
     workspace_api_mock.get_status.return_value = \
         WorkspaceFileInfo('/notebook-name', NOTEBOOK, WorkspaceLanguage.SCALA)
     runner = CliRunner()
-    runner.invoke(cli.export_workspace_cli, ['--format', 'SOURCE', '/notebook-name', path])
+    print(runner.invoke(cli.export_workspace_cli, ['--format', 'SOURCE', '/notebook-name', path]).output)
```

Comment:
What does `print` do here?

Reply:
ah this was for debugging. i need to revert it now.